index.html

```diff
@@ -31,7 +31,7 @@ <h1 class="primary-text">
                     <p>
                         I’m a software developer! I can help you build a product ,
                         feature or website Look through some of my work and experience!
-                        If you like what you see and have a project you need coded,
+                        If you like what you see  and have a project you need coded,
```

Comment:
- [ ] Great job so far 👏🏻 However your images or icons are not showing up correctly 👇🏻 

![19](https://user-images.githubusercontent.com/79058364/185887771-b2606808-38a9-4155-958e-dad01aaead8b.png)


- I think the problem is because of you're missing the `.` in your path, let's add `.` before your image URL, It should be like this: 
-  `<img class="tonic" alt="Portfolio" src="/img/SnapshootPortfolio(1).png" />` (line 53)

- [ ] Kindly make sure your images are showing up correctly 👍🏻 !!

NOTE: Minimum mobile screen size should be `375`px. Otherwise, the mobile version should be responsive until the desktop screen size.